en/ros/README.md

```diff
@@ -13,12 +13,17 @@ The PX4 development team recommend that all users [upgrade to ROS 2](../ros/ros2
 
 ## ROS Setups
 
-PX4 supports both the "original" ROS and ROS 2, with the following configurations:
+PX4 supports both ROS 2 and ROS 1, with the following configurations:
 
 - **[ROS 2](../ros/ros2.md): (Recommended)** PX4 and ROS 2 communicate over the [PX4-ROS 2 bridge](../ros/ros2_comm.md), an interface that provides a direct bridge between PX4 uORB messages and ROS 2 DDS messages/types.
   This effectively allows direct access to PX4 internals from ROS 2 workflows and nodes in realtime.
 - **[ROS 1 via MAVROS](../ros/ros1.md):** PX4 and ROS 1 communicate over [MAVLink](../middleware/mavlink.md), using the [MAVROS](../ros/mavros_installation.md) package to bridge ROS topics to MAVLink.
 
+:::note
```

Comment:
FYI @beniaminopozzan This is one of the two notes I added about MAVROS integration. We might flesh these out later if we want but should cover us for now.